app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+    use Rack::Flash
+    get '/tweets' do
+        if logged_in?
+            @user = User.find_by_id(session[:user_id])
+            erb :'/tweets/tweets'
+        else
+            redirect to "/login"
+        end
+    end
+
+    get '/tweets/new' do
+        if logged_in?
+            erb :'/tweets/create_tweet'
+        else
+            redirect "/login"
+        end
+    end
+
+    post '/tweets' do
+        if !params[:content].empty?
+            @tweet = Tweet.create(:content => params[:content])
+            @tweet.user = current_user
+            @tweet.save
+            @user = @tweet.user
```

Comment:
Was curious as to the use of this line as @user is not being used anywhere else in this controller action or in the view that gets rendered.